litmus-portal/frontend/src/components/WorkflowCluster/styles.ts

```diff
@@ -44,7 +44,12 @@ const useStyles = makeStyles((theme: Theme) => ({
   button: {
     marginLeft: 167,
     textAlign: 'left',
-    marginTop: 64,
+    marginTop: 67,
+  },
+  buttonOutlined:{
+    marginLeft: 359,
```

Comment:
@oumkale You indentation and spacing is off, can you please try running npm run format once before pushing, although husky should handle it but its better you try it manually if husky is not doing it for you